jetty-websocket/websocket-jetty-client/src/main/java/org/eclipse/jetty/websocket/client/WebSocketClient.java

```diff
@@ -68,6 +71,7 @@
     private final Configuration.ConfigurationCustomizer configurationCustomizer = new Configuration.ConfigurationCustomizer();
     private final WebSocketComponents components = new WebSocketComponents();
     private boolean stopAtShutdown = false;
+    private long _stopTimeout = 200;
```

Comment:
Hmmmm I don't really like such hard coded default timeouts.... but at least there is a setter.
What happens if you make the default wait forever?

Reply:
I have changed this default to `Long.MAX_VALUE`, so it will always do the proper close of all the sessions, which is fine assuming it doesn't block in user code in `onClosed`.